<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CProductInstanceInfoViewComp</class>
 <widget class="QWidget" name="CProductInstanceInfoViewComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Product Instance</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="LicenseManagementGroup">
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>License</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="NewLicenseRequestButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Create Transaction Code...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/Icons/DocumentExport</normaloff>:/Icons/DocumentExport</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="LicenseRequestDescriptionLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="LoadLicenseButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Load License...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/Icons/Import</normaloff>:/Icons/Import</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="LicensePathLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="InstanceIdGroup">
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Instance-ID</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="ProductInstanceIdEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="CustomerGroup">
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Customer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="CustomerNameEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ProductSelectionGroup">
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Product</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="ProductNameEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="FeaturesGroup">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>104</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Features</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTreeWidget" name="FeatureTree">
        <property name="indentation">
         <number>0</number>
        </property>
        <attribute name="headerMinimumSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>150</number>
        </attribute>
        <column>
         <property name="text">
          <string>Feature</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>License</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Valid</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
